setup/setup.sh

```diff
@@ -1,9 +1,2 @@
-inst_dir=$( cd $(dirname $BASH_SOURCE)/../..; pwd -P )
-if [ ! -d "$inst_dir/ups_db" ]; then
-   mkdir $inst_dir/ups_db
-fi
-export EUPS_PATH=$inst_dir:${LSSTSW}/stack
-. ${LSSTSW}/bin/setup.sh
-setup lsst_sims
-eups declare Twinkles -r . -c
-setup Twinkles
+# Setup lsst_sims in the way you usually do
+setup Twinkles -t $USER
```

Comment:
This does not work if the user does not have a version of lsst_sims tagged $USER.

I tried running this with a version of eups that only knew about the -t sims version of lsst_sims, it came back with
```
setup: in file /Users/danielsf/physics/lsst_160212/Development/garage/Twinkles/ups/Twinkles.table: Product lsst_sims not found
```
this command
```
setup Twinkles -t $USER -t sims
```
worked.

I think we are going to have to settle for writing a very good README that explains to users how to navigate EUPS, rather than scripting everything for them.  There are too many possible permutations (which is, I guess, why EUPS exists).